index.js

```diff
@@ -41,15 +41,15 @@ export default {
             getParent(el, identifier) {
                 var idn = identifier.toLowerCase();
                 var type = 'class';
-                if (/^#/.test(identifier))  type = 'id';
+                if (/^#/.test(identifier)) type = 'id';
 
-                type == 'class' ?  idn = idn.replace(/./, '') : idn = idn.replace(/#/, '');
+                type == 'class' ? idn = idn.replace(/./, '') : idn = idn.replace(/#/, '');
```

Comment:
I think this regex is wrong. the `.` should be escaped, alternatively just
```
idn = idn.replace(/^\.|#/, '')
```
is probably an easer way to handle both scenarios